hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOMDbCheckpointServlet.java

```diff
@@ -214,6 +225,26 @@ public void testDoGet() throws Exception {
         om.getOmAdminGroups(),
         om.isSpnegoEnabled());
 
+    String crNl = "\r\n";
+    String sstFileName = "sstFile.sst";
+    byte[] data = ("--" +  MULTIPART_FORM_DATA_BOUNDARY + crNl +
+        "Content-Disposition: form-data; name=\"" +
+        OZONE_DB_CHECKPOINT_REQUEST_TO_EXCLUDE_SST + "[]\"" + crNl +
+        crNl +
+        sstFileName + crNl +
```

Comment:
please use 2 sst files.  (Sometimes testing an array of size 1 hides flaws in loop code.)